install_config/configuring_openstack.adoc

```diff
@@ -166,12 +154,11 @@ kubeletArguments:
   cloud-config:
     - "/etc/cloud.conf"
 ----
-<1> Name of the OpenStack instance where the node runs (i.e., name of the virtual machine)
-====
+<1> Name of the OpenStack instance where the node runs (that is, the name of the virtual machine)
```

Comment:
Maybe:
<1> The RFC1123-compliant OpenStack instance name of the node host.

And then a note that says:
If the `nodeName` does not match the OpenStack instance name, the cloud provider integration will not work.